### DISCUSSION TITLE
DSAR REQUEST - Remove Billing Information #132279

### DISCUSSION THREAD

**Discussion by Kaotic3** [community/community#132295] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Thank you for confirming that this space is managed by GitHub Employees:

"This community is managed by GitHub employees"

This DSAR request stands, as stated in the original post - I am not required to do anything special other than alert the organisation to my request.

```
You should be aware of European Legislation that still applies to the UK:
https://ico.org.uk/for-the-public/your-right-to-get-your-data-deleted/

You don’t have to ask a specific person – you can contact any part of the organisation with your request.
```

I have done that and I repeat it below for your convenience.

You have 30-days to comply, at which point I will make a complaint to the ICO in the UK, at which point they will investigate and may issue fines for failure to delete data as requested under both European Union Law and UK law.

We do not have to do anything special for a DSAR request to stand - as you can imagine an unscrupulous company could take advantage of that and make it difficult for people to have their data deleted - that is why the law says I only need to alert the company.

You are employees.

I have told you.

Therefore the company is alerted.

This is my request:

```
DSAR REQUEST - Remove Billing Information

If you can tell me how to do this, I will remove it myself. There doesn't appear to be a way to remove it, just exchange it for another payment choice, as such, I have had to make this a DSAR request.

Please accept this as a DSAR request to delete all billing information that you hold for this account.

You should be aware of European Legislation that still applies to the UK:
https://ico.org.uk/for-the-public/your-right-to-get-your-data-deleted/

You don’t have to ask a specific person – you can contact any part of the organisation with your request.

If you are unable to provide a way for me to delete the billing information myself, could you acknowledge receipt of my DSAR request and that this request is being passed to the correct people.

Thanks,
```

If you close this again without taking action, I will not open another ticket, but if the data is not deleted in 30 days, I will make a complaint and both these requests will stand as information for the commissioner to consider.

**Comment by LiteBrite82** [comment]
Hi @Kaotic3!

Thank you for posting!

This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

The only avenue for support in this matter is through our [Support page](https://support.github.com/contact). Please open a ticket with them if you have not already, please only open 1 ticket per incident. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
